cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -3608,4 +3608,56 @@ TEST_F(TestProjector, TestExtendedFunctions) {
   EXPECT_ARROW_ARRAY_EQUALS(out, outs.at(0));
 }
 
+TEST_F(TestProjector, TestExtendedStubFunctions) {
```

Comment:
This test case demonstrates registering an external function (but this function doesn't use function holder)